txp/Pos/Txp/Settings/Global.hs

```diff
@@ -37,7 +37,6 @@ type TxpCommonMode ctx m =
 type TxpGlobalVerifyMode ctx m =
     ( TxpCommonMode ctx m
     , MonadError ToilVerFailure m
-    , MonadSlots m  -- TODO: I don't like it (@gromak)
```

Comment:
Lol, in the master there isn't this constraint and it works (I suppose).
I just returned this constraint to `master` state, why wouldn't it work?